ovos_core/intent_services/dispatcher.py

```
@@ -0,0 +1,272 @@
+from ovos_utils.log import LOG
+
+#: default upper bound on handler execution before §8.3 timeout fires, seconds.
+#: handlers are long-running by design (§6.5) so this is generous; set to 0 or a
+#: negative value (config ``intents.handler_timeout``) to disable the timer.
+DEFAULT_HANDLER_TIMEOUT = 5 * 60
+
+
+class _InFlightDispatch:
+    """A dispatch awaiting its §8 terminal."""
+
+    __slots__ = ("skill_id", "intent_name", "dispatch_msg", "timer", "resolved")
```

Ruff (0.15.20)

[warning] `_InFlightDispatch.__slots__` is not sorted

Apply a natural sort to `_InFlightDispatch.__slots__`

(RUF023)